types/react-select/src/Select.d.ts

```diff
@@ -161,7 +161,7 @@ export interface Props<OptionType = { label: string; value: string }> extends Se
   /* Name of the HTML Input (optional - without this, no input will be rendered) */
   name?: string;
   /* Text to display when there are no options */
-  noOptionsMessage?: (obj: { inputValue: string }) => string | null;
+  noOptionsMessage?: (obj: { inputValue: string }) => React.ReactNode;
```

Comment:
I'm fine with the `placeholder` change, but I'm not sure I agree with changing the type of `noOptionsMessage`. While it might very well work with any React node right now, there are no guarantees that it will work with a React node in the future since it's not officially stated anywhere that you can return any React node and the Flow types also don't allow for any React node. I would suggest creating a PR on `react-select` to update their [Flow types and documentation](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.js#L201) and then we can update it here.

Reply:
Sure, makes sense. Undid that change